dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -71,7 +71,19 @@ def max_plus_pending(max_resource, pending_resource):
         unit="actors",
         targets=[
             Target(
-                expr="sum(ray_actors{{{global_filters}}}) by (State)",
+                expr='sum(ray_actors{{Source="gcs",{global_filters}}}) by (State)',
+                legend="{{State}}",
+            )
+        ],
+    ),
+    Panel(
```

Comment:
can we just replace the Scheduler Actor State graph completely? Or is it still useful?

Reply:
yes, I think it's still useful.